tests/libcxx/expected_results.txt

```diff
@@ -531,37 +532,37 @@ std/numerics/complex.number/complex.special/double_long_double_implicit.compile.
 std/numerics/complex.number/complex.special/float_double_implicit.compile.fail.cpp FAIL
 std/numerics/complex.number/complex.special/float_long_double_implicit.compile.fail.cpp FAIL
 
-# STL bug: regex_traits::transform() isn't following the Standard.
+# GH-1004 regex_traits::transform() isn't following the Standard.
 std/re/re.traits/transform.pass.cpp FAIL
 
-# STL bug: Incorrect return types.
+# GH-1260 Incorrect return types.
 std/numerics/complex.number/cmplx.over/pow.pass.cpp FAIL
 
-# STL bug: We allow fill() and swap() for array<const T, 0>.
+# GH-1295 <array>: array<const T, 0> allows fill() and swap()

```

Comment:
Copying the GitHub titles makes sense, I'll clean this up.

Reply:
Done; for the "missing locale names" issue, I copied the issue title exactly, and then preserved the existing explanation as a following comment, just in case. (I felt that the others provided equivalent information.)